.travis.yml

```diff
@@ -61,3 +61,16 @@ after_script:
 after_success:
     - travis-sphinx -v --outdir=docbuild deploy
     - codecov
+
+deploy:
+  provider: pypi
+  user: "__token__"
+  password:
+    secure: "NApRRmqvrZDcyY1s7IzlD/V+H6K7zb0Sm7c+tX7v412ymL2X86NxZuBY+f7E+/YhZAjgW1kZGJvuifRRz422o3dg5d4G4u3deswTI+L5//vW+qhF3DJRkGAlMm6RKjILe6ePMQxyg5pK7JC8bFfb+5hQgEXEPBGr16zq68CrB0orDqmD1cSi2ur3MTQO6GOJnB8bW33LnCYSP9nb9PNRTHOAggpcuhET57DGxwnI7Icjq6bCnwOzdAEag9CvCh6dsEDnLgbSbxn1TIACtmZZiA757VE9Aw2xQdRCR9ncBY6Y2PZKrdbznA8xi9WQTB5k7+1r3KBNQv5+k04pYXqL/2dx/67wJdOQTckbW2cp+6kzfRdvvJewqg9mC+Wxrgt828dJWvIrR02mIWXdUxV7sxhiP01LX0nH7yC9VdWS3uYAQv768m6/3HgFuTRD2GNxuk3h89lbpZ8A0+7AcBtMAuhr8gi26tvMVJsilxnEs2L4oPStpvbdz5n3qjPUupyR7ek08umtC5AMCdPORkAqw3y6TNXKaSogGMdOew/K/ZyYRexs95ITUqXm5r6MZ7PcLtUyRU7sENJ/Au9U7k1Bb3e2JH/IEdPGjy2fq1lrJIXnGNZkgxLa8zcQJxL5F7zr6CnUQypqM4mVr+1wxsFGJmwvYdVLmzNIc9fOSX7kt/U="
```

Comment:
Should this be in the code, or as a Travis variable? Seems insecure

Reply:
This is following the travis guidelines. It's encrypted. See https://docs.travis-ci.com/user/deployment/pypi/